ext/libxml/libxml.stub.php

```diff
@@ -172,7 +172,7 @@ function libxml_get_errors(): array {}
 
 function libxml_clear_errors(): void {}
 
-/** @deprecated */
+#[\Deprecated(since: '8.0')]
```

Comment:
@nielsdos Would `libxml_disable_entity_loader(false)` reenable it or not?

Reply:
Yes but only when passing LIBXML_NOENT to the parsing options of the XML extensions.

Reply:
I understand that external entities are dangerous, but the deprecation without any alternative sounds wrong.

Anyway: Do you have a specific wording suggestion here? Best to suggest it using GitHub's suggestion feature.

Reply:
> but the deprecation without any alternative sounds wrong.

You can still use them though, this was necessary to combat shit defaults in libxml2 < v2.9. Since PHP 8.0 we require at least libxml2 v2.9 and so the problem went away.

Reply:
```suggestion
#[\Deprecated(message: 'External entity loading is disabled by default', since: '8.0')]
```

Reply:
> You can still use them though.

I understand this is not the correct place for this discussion, but does it require the use of this deprecated function to enable them or will this happen automatically when calling `libxml_set_external_entity_loader()` or so? If the former, then this doesn't really feel like still being able to use them.

Regarding the change: Thanks, applied (with `s/External/as external/` to better fit the grammar).

Reply:
> I understand this is not the correct place for this discussion, but does it require the use of this deprecated function to enable them 

No you don't have to use this. You have to pass the LIBXML_NOENT option to an XML parser. libxml_set_external_entity_loader() is orthogonal to this